docs/interactive/Accessing State Dependent Properties.ipynb

```diff
@@ -15,11 +15,11 @@
     "dictionary. These are the following strings, which will be replaced during execution by\n",
     "their respective current values:\n",
     "\n",
-    "- `\"%observations.ivs[-1]%\"`: the last observed independent variables\n",
-    "- `\"%observations.dvs[-1]%\"`: the last observed dependent variables\n",
-    "- `\"%observations.ivs%\"`: all the observed independent variables,\n",
+    "- `\"%data.conditions[-1]%\"`: the last observed independent variables\n",
```

Comment:
this conflicts with the [renaming in the other PR](https://github.com/AutoResearch/autora-workflow/pull/15/commits/2b5ed10643b591f8eaf6b6f2baef674a720846ac#:~:text=committed%202%20hours%20ago), no?

Reply:
Good catch!